cugraph-analytics/cugraph_analytics.ipynb

```diff
@@ -0,0 +1,453 @@
+    "\n",
+    "### Memgraph Lab installation\n",
+    "The last piece of tech you will need is **[Memgraph Lab](https://memgraph.com/lab)**.  You will use it to connect to **Memgraph** and create data visualizations. Check out how to install **Memgraph Lab**  as a desktop application for [Windows](https://memgraph.com/docs/memgraph-lab/installation/windows#step-2---installing-and-setting-up-memgraph-lab), [Mac](https://memgraph.com/docs/memgraph-lab/installation/macos#step-2---installing-and-setting-up-memgraph-lab)  or [Linux](https://memgraph.com/docs/memgraph-lab/installation/linux#step-2---installing-and-setting-up-memgraph-lab), and then [connect to Memgraph database](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph).\n",
+    "\n",
+    "With the **Memgraph Lab** installed and connected, we are ready to connect to Memgraph with **GQLAlchemy**, import our large dataset and run graph analytics using **Python**."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Connecting to Memgraph with GQLAlchemy\n",
+    "\n",
+    "The next three lines of code will import `qglalchemy`, connect to **Memgraph** database instance via `host:127.0.0.1` and `port:7687`, and clear the database, just to be sure we are starting with a clean slate."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gqlalchemy import Memgraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "memgraph = Memgraph(\"127.0.0.1\", 7687)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "memgraph.drop_database()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's import the dataset from CSV files and learn how to perform **PageRank** and **Louvain community detection** using **Python**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Importing data\n",
+    "\n",
+    "The **CSV** files containing the [**Facebook** dataset](https://snap.stanford.edu/data/gemsec-Facebook.html) have the following structure:\n",
+    "```\n",
+    "node_1,node_2\n",
+    "0,1794\n",
+    "0,3102\n",
+    "0,16645\n",
+    "```\n",
+    "The dataset consists of verified Facebook pages belonging to different categories and dating back to November 2017. Nodes represent the pages, and relationships are mutual likes among them. The nodes are reindexed (starting from 0) to achieve a certain level of anonymity. Since **Memgraph** imports query faster when data has indexes, we will create them for all the nodes with the label `Page` on the `id` property."
```

Comment:
Clarify index description + change wording
```suggestion
    "The dataset consists of verified Facebook pages belonging to various categories and dating back to November 2017. Each node stands for a page, and relationships represent mutual likes. The nodes are reindexed (starting from 0) to achieve a certain level of anonymity. Because Memgraph imports indexed data faster, we will create an index for `Page` nodes over the `id` property."
```